tests/manage/noobaa/conftest.py

```diff
@@ -0,0 +1,111 @@
+import logging
+
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import noobaa
+from tests import helpers
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def noobaa_obj():
+    """
+    Returns a NooBaa resource that's connected to the S3 endpoint
+    Returns:
+        NooBaa: A NooBaa resource
+
+    """
+
+    noobaa_obj = noobaa.NooBaa()
+    return noobaa_obj
+
+
+@pytest.fixture()
+def uploaded_objects(request, noobaa_obj, awscli_pod):
+    """
+    Deletes all objects that were created as part of the test
+
+    Returns:
+        list: An empty list of objects
+
+    """
+
+    uploaded_objects_paths = []
+
+    base_command = (
+        f"sh -c \"AWS_ACCESS_KEY_ID={noobaa_obj.access_key_id} "
+        f"AWS_SECRET_ACCESS_KEY={noobaa_obj.access_key} "
+        f"AWS_DEFAULT_REGION={noobaa_obj.region} "
+        f"aws s3 "
+        f"--endpoint={noobaa_obj.endpoint} "
+    )
+
+    def object_cleanup():
+        for uploaded_filename in uploaded_objects_paths:
+            # TODO: Add assert
```

Comment:
add assert?

Reply:
My bad, should've removed that. I wanted to add an assertion that verifies the files were removed, but later on, when removing the bucket, an exception should be raised in case it couldn't be deleted (because of leftover files), making this assertion unneeded.

Also, apparently, the AWSCLI doesn't tell you whether the object you're trying to delete even exists. The only way is to list the files manually and make sure it isn't there.